classes/components/CategoryManager.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file classes/components/CategoryManager.php
+ *
+ * Copyright (c) 2014-2025 Simon Fraser University
+ * Copyright (c) 2000-2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class CategoryManager
+ *
+ * @ingroup classes_components
+ *
+ * @brief A component for viewing and managing Categories
+ */
+
+namespace PKP\components;
+
+use APP\core\Application;
+use APP\file\PublicFileManager;
+use PKP\components\forms\context\CategoryForm;
+use PKP\context\Context;
+use PKP\facades\Locale;
+use PKP\facades\Repo;
+
+class CategoryManager
+{
+    public const COMPONENT_CATEGORY = 'CategoryManager';
+    public $id = self::COMPONENT_CATEGORY;
+
+    public function __construct(
+        private Context $context,
+    ) {
+
+    }
+
+    public function getConfig(): array
```

Comment:
The methods are missing the phpdoc, if you have patience, it's nice to document the return with the array shape from PHPStan.